deploy/Dockerfile

```diff
@@ -25,3 +25,7 @@ COPY --from=build /usr/src/app/public/index.html /usr/share/nginx/html/index.htm
 COPY --from=build /usr/src/app/public/build/bundle.js /usr/share/nginx/html/build/bundle.js
 
 RUN chmod a+r -R /usr/share/nginx/html
+
+COPY .docker/docker-entrypoint.sh /
+ENTRYPOINT ["/docker-entrypoint.sh"]
```

Comment:
@Matthew-Kulich @LaChope It was not possible to use nginx envsubst enpoint?

Reply:
@blcham It is not working yet, we are just still under investigation :)

Reply:
oh sry ;)